src/content/capture/canvas-record/js/main.js

```diff
@@ -31,15 +31,7 @@ recordButton.onclick = toggleRecording;
 playButton.onclick = play;
 downloadButton.onclick = download;
 
-// Call main() in demo.js to init teapot
-main();
-
-// window.isSecureContext could be used for Chrome
-let isSecureOrigin = location.protocol === 'https:' || location.hostname === 'localhost';
-if (!isSecureOrigin) {
-  alert('getUserMedia() must be run from a secure origin: HTTPS or localhost.\n\nChanging protocol to HTTPS');
-  location.protocol = 'HTTPS';
-}
+document.addEventListener('DOMContentLoaded', main);
```

Comment:
Is this necessary, given that the script element is at the bottom of the page?

(Or am I missing something...)

Reply:
Not really. I think this ended up here because I was doing something similar in a different place and my brain is bad at multi-tasking. :)

Reply:
Removed it.